### PR TITLE
[MFTF] Used In information is not displayed after the entity was deleted

### LINKED ISSUE #1774: [MFTF] Used In information is not displayed after the entity was deleted
Test case link: https://studio.cucumber.io/projects/131313/test-plan/folders/1320712/scenarios/4885068

### Preconditions (*)
1. Admin is logged in
2. A Product, Category, Block and Page are created

### Steps
1. Upload an image to the media gallery
2. Add the image to the created Product, Category, Block and Page
3. View details of the image from the Media Gallery and verity that "Used In" section contains all 4 entities
4. Delete all 4 entities
5. View details of the image from the Media Gallery and verity that "Used In" section is not displayed

### PR DESCRIPTION
**Pull request by Nazar65**
deleted with mftf tests

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1774: Used In information is not displayed after the entity was deleted

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AssertAdminEnhancedMediaGalleryUsedInSectionNotDisplayedActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminEnhancedMediaGalleryUsedInSectionNotDisplayedActionGroup">
+        <annotations>
+            <description>Assert thats used in section not displayed in vew details.</description>
```
Comment (BarnyShergold) [inline app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AssertAdminEnhancedMediaGalleryUsedInSectionNotDisplayedActionGroup.xml]: Typos in messages

```suggestion
            <description>Assert that's used in section not displayed in view details.</description>
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @Nazar65 ! It would be good to add similar tests for other entities (product, block, page) to cover all the observers

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**